src/tlo/methods/healthsystem.py

```diff
@@ -2737,6 +2880,17 @@ def write_to_log_and_reset_counters(self):
                 "Number_By_Appt_Type_Code_And_Level": self._never_ran_appts_by_level,
             },
         )
+            # Log summary of HSI_Events that do not have a blank appointment footprint
```

Comment:
Re-align comment to be inline with logger_summary.info(
```suggestion
            # Log summary of HSI_Events that do not have a blank appointment footprint
            logger_summary.info(
```